Libraries/LibWeb/Layout/TreeBuilder.h

```diff
@@ -64,6 +67,31 @@ class TreeBuilder {
     Vector<GC::Ref<Layout::NodeWithStyle>> m_ancestor_stack;
 
     u32 m_quote_nesting_level { 0 };
+
+    // Everything we need to recalculate `content` that contains `reversed` counters for a pseudo-element.
+    struct ElementWithReversedCounterContent {
+        GC::Ptr<DOM::Element> element;
+        CSS::PseudoElement pseudo_element;
```

Comment:
Could this hold a `DOM::AbstractElement` instead? I'm trying to move away from passing around an Element and PseudoElement separately.